k8s/kubevirt-dynamic/virtualmachineinstancemigration.go

```diff
@@ -61,6 +69,14 @@ type VirtualMachineInstanceMigrationOps interface {
 func (c *Client) CreateVirtualMachineInstanceMigration(
 	ctx context.Context, vmiNamespace, vmiName string,
 ) (*VirtualMachineInstanceMigration, error) {
+	return c.CreateVirtualMachineInstanceMigrationWithParams(ctx, vmiNamespace, vmiName, "", "", nil, nil)
+}
+
+// CreateVirtualMachineInstanceMigrationWithParams starts live migration of the specified VMI
+func (c *Client) CreateVirtualMachineInstanceMigrationWithParams(
+	ctx context.Context, vmiNamespace, vmiName string, migrationName, generateNamePrefix string,
```

Comment:
Curious how is generatedNamePrefix going to be used? 
In what cases will we not provide migrationName but provide an explicit generateNamePrefix?
Also this is generated name of what exactly?

Reply:
k8s allows you to use "generateName" in the spec instead of "name" for the object. It is used when you don't care about the exact name and just want some unique name for the obj.
https://kubernetes.io/docs/reference/using-api/api-concepts/#generated-values